pricegraph/src/orderbook.rs

```diff
@@ -798,6 +854,65 @@ mod tests {
         assert_eq!(orderbook.num_orders(), 0);
     }
 
+    #[test]
+    fn fills_order_at_price_with_correct_amount() {
+        //    /-1.0---v
+        //   /--2.0---v
+        //  /---4.0---v
+        // 1          2
```

Comment:
Not sure what this represents.. orders buying 1, 2 and 4 of Token1 for the same amount of Token2. Its a cute illustration, but I am not sure I am getting the intended intuition here.

Reply:
The illustration represents the graph representation of the orderbook, where the `1` and `2` are token nodes, and the arrows are orders, with their weights being prices.

Reply:
AHH! I see with the little arrows! Noice (as above).